doc/rst/technotes/libraries.rst

```diff
@@ -384,6 +416,68 @@ For example:
    Make sure your Python functionality is also complete before calling this
    function.
 
+.. note::
+
+  If you are taking advantage of the generated ``__init__.py`` initializer
+  file to import the output directory as a package, you do not need to call
+  ``chpl_cleanup()`` yourself because it is already registered to be called
+  at program exit. The generated initializer is explained below.
+
+Python Init File
+----------------
+
+A simple ``__init__.py`` file is generated in the output directory along with
+the Python module. It looks roughly like the following:
```

Comment:
Good call on the use of roughly, given that this can change over time.

Reply:
Thanks!